src/api/app/assets/stylesheets/webui/user_profile_redesign/user_profile.scss

```diff
@@ -1,6 +0,0 @@
-body.user-profile-redesign {
```

Comment:
File moved to *src/api/app/assets/stylesheets/webui/user_profile.scss*, but without `body.user-profile-redesign { ... }`